unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Camera/OverrideCinemachineAxisInput.cs

```diff
@@ -32,6 +33,10 @@ private float OverrideGetAxis(string axisName)
             return 0;
         
         float value = cachedAxisToMeasurableActions[axisName].GetValue();
+        if (axisName.Equals("Mouse Y") && invertMouseY) 
```

Comment:
Can you please store these kind of hardcoded strings ("Mouse Y") into constants? Thanks!